helm/charts/hpe-csi-driver/README.md

```diff
@@ -1,60 +1,65 @@
-# HPE CSI Volume Driver Helm Chart
+# HPE CSI Driver for Kubernetes Helm chart
+The [HPE CSI Driver for Kubernetes](https://github.com/hpe-storage/csi-driver) leverages HPE storage platforms to provide scalable and persistent storage for stateful applications.
 
-## Introduction
-
-The Container Storage Interface (CSI) Volume Driver for Kubernetes leverages HPE Nimble Storage to provide scalable and persistent storage for stateful applications.
+## Prerequisites
+- Upstream Kubernetes version > 1.10
+- Other Kubernetes distributions supported
+  - OpenShift 4.1 in Tech Preview (3.x is not supported, see [FlexVolume Helm chart](../hpe-flexvolume-driver))
+  - More distributions will be listed as tests are ongoing
+- Recent Ubuntu, CentOS or RHEL compute nodes connected to their respective official package repositories
 
-## Features list
+Depending on which [Container Storage Provider](https://github.com/hpe-storage/container-storage-provider) (CSP) is being used, other prerequisites and requirements may apply.
 
-- Persistent Volumes
-- Snapshots with Replication
-- Online Volume Expansion
-- Import Volumes
+### HPE Nimble Storage CSP
+- NimbleOS 3.9 or later
```

Comment:
4.6 or later

Reply:
What happened to "all the way to Daytona Beach?" 🏖

Reply:
I said, "All the way to Emerald Isle."